src/scripts/updateOED.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# *
+# * This Source Code Form is subject to the terms of the Mozilla Public
+# * License, v. 2.0. If a copy of the MPL was not distributed with this
+# * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# *
+
+echo "Upgrading OED"
+
+# Install NPM dependencies
+echo "NPM install..."
+npm install --loglevel=warn --progress=false
+echo "NPM install finished."
+
+create_error=0 # Boolean
+
+tries=0
+max_tries=10
+
+# Try to create the schema until it succeeds
+while [ $create_error == 0 ]; do
+    # Sleep to let PostgreSQL chill out
+    sleep 1
+    echo "Attempting to create database."
+    # Redirect stderr to a file
+    npm run createdb |& tee /tmp/oed.error > /dev/null
```

Comment:
I remembered when I tried to run createdb, it keeps saying that meter_type already exists. I don't know how to override that

Reply:
This is alright with that; it should just emit "Schema created or already exists" and move on.